src/ast_parser.rs

```diff
@@ -71,19 +59,12 @@ impl fmt::Display for SwcDiagnostic {
 }
 
 impl SwcDiagnostic {
```

Comment:
We could maybe remove this and just use the `deno_ast` diagnostic?

Reply:
Sounds good to me, feel free to do that in a follow up